src/axiosHelpers/axiosDeleteHelper.ts

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { AxiosResponse } from "../app/models/AxiosResponse";
+import { ACCESS_TOKEN, BASE_URL } from "../constants";
+
+let config = {
```

Comment:
Move config to globals constants.ts file, so its not redundant and reusable on all axiosHelpers.